src/serviceFacades/filesystem.js

```diff
@@ -286,3 +286,18 @@ export const pathListCreator = ({
         params,
     });
 };
+
+export const restore = (paths) => {
+    return callApi({
+        endpoint: "/api/filesystem/restore",
+        method: "POST",
+        body: [paths],
```

Comment:
Looking at the [API](https://cyverse-de.github.io/api/endpoints/filesystem/restore.html#restoring-a-file-or-directory-from-a-users-trash), the body is `{ paths: [paths] }`.  Does this work too?

Reply:
Actually it works the way as it is (although am not sure why it works) but to simplify it can be just `body: paths`.